sdk/storage/storage-blob/tests.yml

```diff
@@ -9,6 +9,12 @@ resources:
       name: Azure/azure-sdk-tools
       endpoint: azure
 jobs:
-  - template: ../archetype-sdk-tests-storage.yml
+  - template: ../../../eng/pipelines/templates/jobs/archetype-sdk-integration.yml
     parameters:
       PackageName: "@azure/storage-blob"
+      ResourceFileLocation: sdk/storage/storage-blob/tests-resources.json
+      EnvVars:
+        ACCOUNT_NAME: $(aad-storage-account-name)
+        ACCOUNT_KEY: $(aad-storage-account-key)
+        ACCOUNT_SAS: $(aad-azure-sdk-test-account-sas)
+        CONNECTION_STRING: $(aad-azure-storage-connection-string)
```

Comment:
These variables are not static. They are specific to the values generated by the ARM templates. They should be named based on the outputs from the `test-resources.json`: 

```
ACCOUNT_NAME: $(job-storage-account-name)
ACCOUNT_KEY: $(job-storage-account-key)
ACCOUNT_SAS: $(job-storage-account-sas)
CONNECTION_STRING: $(job-storage-connection-string)
```